braindecode/preprocessing/preprocess.py

```diff
@@ -108,7 +113,8 @@ def __init__(self, fn, channel_wise=False, **kwargs):
                          **kwargs)
 
 
-def preprocess(concat_ds, preprocessors):
+def preprocess(concat_ds, preprocessors, save_dir=None, overwrite=False,
+               n_jobs=None):
```

Comment:
I don't see a benefit in accepting `n_jobs=None` here. We could set default to 1, right?

Reply:
My understanding is that if you provide `n_jobs=None` to joblib it can use the value set with a parallel_backend context manager: https://joblib.readthedocs.io/en/latest/generated/joblib.Parallel.html By default this will be interpreted as 1.